src/add-search-spec.js

```diff
@@ -1,6 +1,7 @@
+'use strict';
+
 const path = require('path');
 const fs = require('fs');
-const config = require(path.join(process.cwd(), 'skyuxconfig.json'));
 
 const template = `// Use browser to access other sites (that are running angular)
 import { browser, element, by } from 'protractor';
```

Comment:
This stuff is so cool! Not important now, but I wish there was a way to test this somehow...

Reply:
Yeah. :/ The only way I know of is to just do it manually in a Stache project. That's how I've been testing it to this point.

Reply:
It works though. Pinky promise.